src/register/register-gnome/gnucash-sheet-private.c

```diff
@@ -423,42 +423,41 @@ draw_cell (GnucashSheet *sheet,
     /* top */
     draw_cell_line (cr, bg_color,
                     (borders.top >= borders.left ? x : x + 1.0),
-                    y + 0.5,
+                    (y == 0 ? y + 0.5 : y - 0.5),
```

Comment:
Changed this to always subtract 0.5 from the y axis so all rows should be the same. The first row uses the header border for the top border.